### PR TITLE
fix error while docker-compose

### LINKED ISSUE #281: Error while executing: docker-compose up / down
While trying to execute `docker-compose up` or `docker-compose down` an error occurs:
ERROR: Error while attempting to convert service.yaptide_worker.deploy.resources.limits.cpus to appropriate type: "/1" is not a valid float

### PR DESCRIPTION
**Pull request by kacper1409**
changed env variable syntax in docker-compose.yml

### CODE REVIEW

**Comment by grzanka** [comment]
Is there an issue related to this PR ?

**Comment by kacper1409** [comment]
> Is there an issue related to this PR ?

Yes, there is #281 . Should I include it in title or description?

**Comment by grzanka** [comment]
> > Is there an issue related to this PR ?
> 
> Yes, there is #281 . Should I include it in title or description?

I've added it here:
image
then merging this PR would fix the issue.

File: docker-compose.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ services:
         deploy:
             resources:
                 limits:
-                    cpus: ${MAX_CORES-/1}
+                    cpus: ${MAX_CORES-1.0}
```
Comment (Bogdan2423) [inline docker-compose.yml:34]: add a comment explaining how this syntax works (provide a link to a website explaining variable expansion with default values)

**Review by Bogdan2423 — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (kacper1409) [reply docker-compose.yml:34]: I think I found something here: https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Comment (grzanka) [reply docker-compose.yml:34]: Good, then add a comment in the `yml` file

**Review by Bogdan2423 — APPROVED** [review]

**Pull request merged**